Sources/Instructions/Core/Public/CoachMark.swift

```diff
@@ -33,6 +33,7 @@ public struct CoachMark {
     /// The path to cut in the overlay, so the point of interest will be visible.
     public var cutoutPath: UIBezierPath?
 
+    public var extraCutoutPath : [UIBezierPath]?
```

Comment:
Seems like a WIP, so the property and its related code found throughout the diff should not be included in this PR.

Reply:
yes, the extraCutoutPath used when we need more than one highlighted area. sorry about that ,i will add example code and  make another PR for that .